backends/bigquery/sql/3_02_vpc_flows_logging_disabled.sql

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+SELECT
+  receiveTimestamp, timestamp AS eventTimestamp,
+  protopayload_auditlog.requestMetadata.callerIp,
+  protopayload_auditlog.authenticationInfo.principalEmail, 
+  protopayload_auditlog.resourceName,
+  protopayload_auditlog.methodName
+FROM `[MY_PROJECT_ID].[MY_DATASET_ID].cloudaudit_googleapis_com_activity`
+WHERE
+protopayload_auditlog.methodName = "v1.compute.subnetworks.patch" 
+AND protopayload_auditlog.request.enableFlowLogs = "false"
```

Comment:
I don't think this SQL query works since `request` field name is converted to `requestJson` as part of log sink to BigQuery [1]. 
Here's a working version you can use.
```
SELECT
  timestamp,
  protopayload_auditlog.requestMetadata.callerIp,
  protopayload_auditlog.authenticationInfo.principalEmail, 
  protopayload_auditlog.resourceName as subnetName,
  protopayload_auditlog.requestJson,
  protopayload_auditlog.methodName,
FROM `[MY_PROJECT_ID].[MY_DATASET_ID].cloudaudit_googleapis_com_activity`
WHERE
  protopayload_auditlog.methodName = "v1.compute.subnetworks.patch" 
  AND JSON_VALUE(protopayload_auditlog.requestJson, "$.logConfig.enable") = "false"
```

[1] https://cloud.google.com/logging/docs/export/bigquery#audit-logs

Reply:
Fixed in commit: [68812e2d098928ccfbf0da5136b7591bccd876df](https://github.com/GoogleCloudPlatform/security-analytics/pull/11/commits/68812e2d098928ccfbf0da5136b7591bccd876df)